### DISCUSSION TITLE
Rainbow Dye and Saved items

### DISCUSSION THREAD

**Discussion by hotspuraz** [Zrips/CMI#7415] [Q&A]
Hello

I found this post regarding Rainbow Dyed items https://github.com/Zrips/CMI/issues/2938, and zrips suggested that you could use /saveditems to store and give items with this dye applied to anyone. However, it doesn't seem to work, as the rainbow dye is soulbound and will only work for the original creator. 

So is there a way to create a piece of leather with the rainbow dye applied that will work for anyone?

**Comment by CloudeLecaw** [comment]
yes
save them as kit...

well i never saved them as single item.. im confused a bit why it does not work there too... might be a bug?
